### PR TITLE
Logo, favicons

### LINKED ISSUE #83: Add 2024 logo

### PR DESCRIPTION
**Pull request by Kriesse**
closes #83 

image

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         Coworking Festival
       </span>
       &bull;
-      <a class="link--footer pl2 pr0" href="/de/impressum">Imprint</a>
+      <a class="link--footer pl2 pr0" href="/de/impressum">Impressum</a>
```
Comment (thilo) [inline _includes/footer.html:32]: its the english footer, so it should stay english. (we don't translate the imprint)

File: _includes/header.html [hunk 2]
```diff
@@ -2,10 +2,14 @@
   <div class="container">
     <a class="logo" href="/">
       <div class="logo__2023">
-        <!-- <img src="{{ site.baseurl }}/img/logo-coworkingfestival-2023.svg" alt="2023" /> -->
+        <img src="{{ site.baseurl }}/img/logo-coworkingfestival-2024.svg" alt="2024" />
```
Comment (thilo) [inline _includes/header.html:5]:
```suggestion
        <img src="{{ site.baseurl }}/img/logo-coworkingfestival-2024.svg" alt="Coworking Festival Logo 2024" />
```

**Review by thilo — APPROVED** [review]
Zwei kleine Sachen. Siehe Kommentare.

File: _includes/header.html (hunk 2 above)
Comment (Kriesse) [reply _includes/header.html:5]: It's intended to just say 2024 – that is a better experience for both screenreaders and in cases where the logo is not loading. As the logo says just "2024", the fallback text/screenreader will read exactly that.

**Pull request merged**